packages/dds/tree/src/feature-libraries/chunked-forest/chunkTree.ts

```diff
@@ -197,6 +197,13 @@ export function chunkFieldSingle(
 	policy: ChunkCompressor,
 ): TreeChunk {
 	const chunks = chunkField(cursor, policy);
+	return combineChunks(chunks);
+}
+
+/**
+ * Combine multiple TreeChunks into a single TreeChunk.
+ */
+export function combineChunks(chunks: TreeChunk[]): TreeChunk {
```

Comment:
chunkFieldSingle was lacking tests previously, which would be nice to fix. Now though we have even more APIs (adding combineChunks) which lack tests.

It would be good to add some basic tests for both of these functions.

Reply:
I would prefer to do this in a separate PR. This change has been in progress for quite some time already. I will do that as a quick follow up. Created AB#51049 to track.